server/src/expressServer.ts

```diff
@@ -1,29 +1,33 @@
-import { middleware } from 'express-openapi-validator';
-import swaggerUI from 'swagger-ui-express';
-import express, { Express } from 'express';
-import cookieParser from 'cookie-parser';
 import bodyParser from 'body-parser';
-import jsYaml from 'js-yaml';
+import cookieParser from 'cookie-parser';
 import cors from 'cors';
+import express, { Express } from 'express';
+import { middleware } from 'express-openapi-validator';
+import fs from 'fs';
 import http from 'http';
+import jsYaml from 'js-yaml';
 import path from 'path';
-import fs from 'fs';
-
-import logger from './logger';
+import swaggerUI from 'swagger-ui-express';
 import config from './config';
+import logger from './logger';
+import { Logger, log } from 'winston';
 
 export default class ExpressServer {
   schema: any;
   app: Express;
   server: http.Server;
+  logger: Logger
 
   constructor(private port: number, private openApiPath: string) {
     this.app = express();
+    this.logger = logger
```

Comment:
One of my teammates suggested we should use a singleton for the logger on the Server. Thoughts?

Reply:
Yes, sure, that's much better